pom.xml

```diff
@@ -98,7 +98,7 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
             <version>2.1.2</version>
-            <scope>test</scope>
+            <optional>true</optional>
```

Comment:
Optional is not a good plan. It complicates classloading. Should just be an extension plugin